CHANGELOG.md

```diff
@@ -6,11 +6,15 @@ All notable changes to this project will be documented in this file based on the
 ### Backward Compatibility Breaks
 
 ### Bugfixes
+* Always set the Guzzle `base_uri` to support connecting to multiple ES hosts. [#1618](https://github.com/ruflin/Elastica/pull/1618)
 
 ### Added
 
+* Added `ParentAggregation` [#1616](https://github.com/ruflin/Elastica/pull/1616)
+
 ### Improvements
 * Added `native_function_invocation` CS rule [#1606](https://github.com/ruflin/Elastica/pull/1606)
+* Elasticsearch test version changed from 6.5.2 to 6.6.1 [#1620](https://github.com/ruflin/Elastica/pull/1620)
```

Comment:
Seems like a bit more changes in the changelog then there should be?

Reply:
Sorry this is my first PR, I'm still discovering how to use git in such way...
I did a rebase of my parent-agg branch onto your origin/master branch but I don't really understand what happened after (my IDE added an empty merge, I don't know why, but I definitely did a rebase). When I try to rebase again, it says that my branch is up-to-date with your master, and for me it seems that way, I suppose I don't understand well what this rebase should have done.
Can you explain me what's wrong and how to fix it please? Really sorry for the inconvenience.

Reply:
One thing that potentially might have happened, is that your master is not up-to-date. Can you go to your master and pull the most recent version? You can use `git log` to see which commit your master is on. Today it should be https://github.com/ruflin/Elastica/commit/d0433d6e6deec8b1547cb4042d97042dcf00ebad (the id or the prefix of it you should see in your git log output on the top).

Reply:
Actually, the first time I tried to rebase directly on your master, not mine (which was not sync with yours).
I tried to redo the operation with my master, after setting it in sync with yours.
It looks good to me this time, I hope it is :-)